src/Runner.cpp

```diff
@@ -128,43 +129,55 @@ void Runner::run(const string& input_path, const string& output_path) {
 
     uniform_real_distribution<double> dis(0.0, 1.0);
     size_t transform_index;
-    size_t batch_idx;
-    auto log_path = fs::path(output_path) / fs::path("log.txt");
+    size_t batch_idx = 0;
     ofstream log_stream;
-#pragma omp parallel \
+
+    #pragma omp parallel \
         private(transform_index, batch_idx) \
         shared(num_files, rewritable_cpaths, option_parsers, descr_per_transform)
     {  // NOLINT
 #pragma omp for
         for (transform_index = 0; transform_index < n_transformations; ++transform_index) {
+            auto log_path = fs::path(output_path) / fs::path("log_" + to_string(transform_index + 1) + ".txt");
```

Comment:
Can you explain, how are you logging the transformations?

Reply:
Since we can have multiple transformations running in parallel (the num of transformations is defined by the `n_transformations` argument), we need to write their logs to separate files. The name of the file where to store logs for `i`-th transform is denoted by `logs_{i+1}.txt`